src/tests/test_raft_repl_dev_dynamic.cpp

```diff
@@ -0,0 +1,132 @@
+/*********************************************************************************
```

Comment:
This test creates repl dev group with 3 replica's by default and then add spare replica's.